lib/shelley/src/Cardano/Wallet/Shelley/Launch.hs

```diff
@@ -1648,6 +1715,8 @@ instance ToText ClusterLog where
         MsgGenOperatorKeyPair dir ->
             "Generating stake pool operator key pair in " <> T.pack dir
         MsgCLI args -> T.pack $ unwords ("cardano-cli":args)
+        MsgListenAddress addr ->
+            "Wallet backend server listening on " <> T.pack (show addr)
```

Comment:
For cardano-wallet:integration there is a separation between `TestsLog` and `ClusterLog`, where `MsgListenAddress` would make sense in `TestsLog`, but _not_ `ClusterLog`.

We could add a similar `TestsLog` for this executable in a follow-up PR.

Reply:
Hmmm. Possibly. Although this one should really be in the application logs of the wallet.